289 rethink signatures

thegnuu opened the pull request.

LevinWi committed:
Removed and applied correct type to signatures

LevinWi committed:
Minor fixes

LevinWi committed:
Minor fixes

thegnuu committed:
feat(draw-overlay): new generalized overlay to add elements

thegnuu committed:
merge(dev): merged dev branch

thegnuu committed:
fix(deepsource): fixed issues

pascalknecht commented on src/app/draw-dialog/draw-dialog.component.html:
```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <div class="noSignature">
```
Klasse entfernen falls nicht benötigt. Sonst mit kebab-case

thegnuu committed:
fix(comments): removed unused comments

deepsource-autofix[bot] committed:
style: format code with Prettier

This commit fixes the style issues introduced in 2a8f717 according to the output
from Prettier.

Details: https://github.com/zskarte/zskarte-client/pull/337

thegnuu committed:
test(playwright): fixed tests

thegnuu committed:
Merge branch '289-rethink-signatures' of github.com:zskarte/zskarte-client into 289-rethink-signatures

thegnuu committed:
fix(deepsource): exclusions

thegnuu committed:
fix(ui): removed unused title on draw overlay
